test/test.js

```diff
@@ -797,6 +797,35 @@ test('HTTP method is case insensitive', (t) => {
   })
 })
 
+const FormData = require('form-data')
```

Comment:
Can you move this to the top of the file, and add form-data to devDependencies?

Reply:
ops... sorry! done